docs/config-example.md

```diff
@@ -12,6 +12,17 @@ authorization_url =
 port =
 plugins = plugin1,plugin2,plugin3
 
+[persistence]
+persistence_name =
+persistence_ip =
+persistence_port =
+local_database_path =
+
+[kubejobs]
+k8s_conf_path = 
+redis_ip = 
+count_queue = 
+
```

Comment:
Explain better what is optional and what should be explicit here. For both options.